source/_integrations/integration.markdown

```diff
@@ -22,10 +22,7 @@ This integrations provides the [Riemann sum](https://en.wikipedia.org/wiki/Riema
 of the values provided by a source sensor. The Riemann sum is an approximation
 of an **integral** by a finite sum.
 
-The integration sensors are updated upon changes of the **source**. Fast
-sampling source sensors provide more accurate results. In this implementation, the
```

Comment:
I would not remove this information as it still applies!

Reply:
What is a "fast sampling source sensor"? And how does it provide "more accurate results"? And more accurate than what?

Reply:
fast sampling means you have more samples of the underlying continuous function (the real physical phenomena), therefore a rienmann sum will be able to better approach the continuous function (regardless of method) if you have more samples.